xcell/mappers/mapper_base.py

```diff
@@ -32,9 +39,31 @@ def _get_rotator(self, coord_default):
             rot = None
         return rot
 
-    def get_signal_map(self):
+    def _get_signal_map(self):
         raise NotImplementedError("Do not use base class")
 
+    def get_signal_map(self, **kwargs):
+        if self.signal_map is None:
+            fn = '_'.join([f'{self.map_name}_signal_map' + (f'_bin{self.zbin}'
+                                                            if self.zbin is not
+                                                            None else ''),
```

Comment:
I'd rather add this to `self.map_name` in the mappers that take redshift bins

Reply:
Yes, I see. But I think it's not a huge deal. I'd rather do that than assume that we always have a `zbin`